### PR TITLE
Fix stretched menu icons

### PR DESCRIPTION
**Pull request by vishalmry**
Issue #14

### CODE REVIEW

File: goa/Classes/menu/MainMenuHome.cpp [hunk 1]
```diff
@@ -78,6 +78,9 @@ bool MainMenuHome::init()
     //    }
 
     cocos2d::ui::Button *backButton = createBackButton();
+    Size frameSize = Director::getInstance()->getOpenGLView()->getFrameSize();
+    float aspectRatio = (1.0 * frameSize.width) / frameSize.height;
+    if (aspectRatio >= 2.0) backButton->setScaleX(0.70);
```
Comment (ankitk20) [inline goa/Classes/menu/MainMenuHome.cpp]: Format this line

File: goa/Classes/menu/MainMenuHome.cpp [hunk 2]
```diff
@@ -78,6 +78,9 @@ bool MainMenuHome::init()
     //    }
 
     cocos2d::ui::Button *backButton = createBackButton();
+    Size frameSize = Director::getInstance()->getOpenGLView()->getFrameSize();
```
Comment (ankitk20) [inline goa/Classes/menu/MainMenuHome.cpp:81]: Can you create a public static variable for _frameSize_ and _aspectRatio_?

**Review by ankitk20 — CHANGES_REQUESTED** [review]

File: goa/Classes/menu/MainMenuHome.cpp (hunk 2 above)
Comment (vishalmry) [reply goa/Classes/menu/MainMenuHome.cpp:81]: I had tried that making the variables global but some of the files are linked producing a redeclaration error during the compilation.

**Review by ankitk20 — APPROVED** [review]

File: goa/Classes/menu/MenuContext.cpp [hunk 3]
```diff
@@ -158,6 +158,17 @@ bool MenuContext::init(Node* main) {
 //    _label = Label::createWithTTF("Points: 0", "fonts/Chanakya.ttf", 150);
 //    _label->setPosition(Vec2(125, 125));
 //    _menuButton->addChild(_label);
+
+    Size frameSize = Director::getInstance()->getOpenGLView()->getFrameSize();
```
Comment (mathewnik90) [inline goa/Classes/menu/MenuContext.cpp:162]: Can we create a common function for this?

**Review by mathewnik90 — APPROVED** [review]

**Pull request merged**